### PR TITLE
[All Hosts] (Dialog) Updating note.

### PR DESCRIPTION
**Pull request by Reezaali**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[86e8a1e](https://github.com/OfficeDev/office-js-docs-pr/pull/922/commits/86e8a1ee2d45f564f60a73de7bddf7644c2ddf6c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/debug-office-add-ins-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/Reezaali-patch-1/docs/testing/debug-office-add-ins-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904251922004295-922%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=nGY5PzyPHZQmijjblYGcbVV%2F0AfBYcZSOOrdJRJ5LRk%3D&st=2019-04-25T19%3A18%3A14Z&se=2019-05-26T19%3A23%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ To start, open a terminal and set the `OfficeWebAddinDeveloperExtras` property f
 
 Then, open the Office application and [sideload your add-in](sideload-an-office-add-in-on-ipad-and-mac.md). Right-click the add-in and you should see an **Inspect Element** option in the context menu.  Select that option and it will pop the Inspector, where you can set breakpoints and debug your add-in.
 > [!NOTE]
-> If you are trying to use the inspector and the dialog flickers, try the following workaround:
+> If you are trying to use the inspector and the dialog flickers, try updating your version of Office. If that doesn't resolve the flicker, try the following workaround:
```
Comment (AlexJerabek) [inline docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: Consider changing "try updating your version of Office." -> "update Office to the latest version."

Comment (AlexJerabek) [inline docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: I think "the flickering" is the proper form here.

Comment (Rick-Kirkham) [reply docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: +1

Comment (kbrandl) [reply docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: +1

Comment (kbrandl) [reply docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: +1 to Alex's feedback here and above as well.  Suggest changing this note to:

> If you are trying to use the inspector and the dialog flickers, update Office to the latest version. If that doesn't resolve the flickering, try the following workaround:

**Comment by kbrandl** [comment]
@Reezaali do you want to incorporate feedback as you see fit, and then merge this PR?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4896d1e](https://github.com/OfficeDev/office-js-docs-pr/pull/922/commits/4896d1e142914cb72a1aff709be0800afda05b5c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/debug-office-add-ins-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/Reezaali-patch-1/docs/testing/debug-office-add-ins-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904291649446398-922%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DkjuOyMKoB3gdtRKrlwLH9LBv9t%2BYN59oEDL%2Fdfhyd0%3D&st=2019-04-29T16%3A45%3A57Z&se=2019-05-30T16%3A50%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**